### ISSUE TITLE
check what we can do with uintptr

### ISSUE THREAD

**Issue by ccoVeille** [ccoVeille/go-safecast#17]

**Comment by ccoVeille** [comment]
As spotted here

- https://github.com/authzed/zed/pull/574

I should support uintptr

https://github.com/authzed/zed/blob/022f31a1892a8612e78ab2a5e902b56a99602539/internal/cmd/schema.go#L226

**Comment by ldemailly** [comment]
@tstirrat15 I have the same annoyance with fd and terminal, it does work with my safecast

https://github.com/fortio/terminal/blob/main/terminal.go#L75-L76

```go
	t.fd = safecast.MustConv[int](os.Stdin.Fd())
	t.fdOut = safecast.MustConv[int](os.Stdout.Fd())
```

can you link the issue you opened with x/term ?

**Comment by ccoVeille** [comment]
uintptr is now supported as any other types in the v2

- #120

**Comment by tstirrat15** [comment]
Yeah, I didn't get around to making the docs change because I never took the time to figure out submitting to the golang core repo.
